pgrx/src/datum/datetime_support/mod.rs

```diff
@@ -24,6 +24,9 @@ mod ops;
 
 pub use ctor::*;
 
+pub const USECS_PER_SEC: i64 = 1_000_000;
+pub const USECS_PER_DAY: i64 = pg_sys::SECS_PER_DAY as i64 * USECS_PER_SEC;
```

Comment:
@eeeebbbbrrrr, why does `pg_sys` has `SECS_PER_DAY` generated but not `USECS_PER_SEC`?

Reply:
hmm.  Probably b/c of how it's defined in `timestamp.h`:

```c
#define USECS_PER_SEC	INT64CONST(1000000)
```

Kinda doubt bindgen can see through even that simple of a #define.